src/components/PlatformOnboardingAccounts/PlatformOnboardingAccounts.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useContext } from 'react'
+import { PlatformOnboardingContext } from '../../contexts/PlatformOnboardingContext'
+import Link from '../../icons/Link'
+import { Button } from '../Button'
+import { Heading, HeadingSize, TextSize, Text } from '../Typography'
+
+export const PlatformOnboardingAccounts = () => {
+  const { nextStep } = useContext(PlatformOnboardingContext)
+  return (
+    <div className='Layer__platform__onboarding__accounts-wrapper'>
+      <div className='Layer__platform__onboarding__header'>
+        <Heading size={HeadingSize.secondary}>
+          Let’s connect the bank accounts and credit cards you use for your
+          business.
+        </Heading>
+      </div>
+      <div className='Layer__platform__onboarding__connect-accounts'>
+        <div className='Layer__platform__onboarding__connect-account-icon'></div>
```

Comment:
Perhaps we could use Plaid icon here for now:

![image](https://github.com/user-attachments/assets/bb1645cb-d96f-40d2-a4b0-476cd14abaee)